src/gosling-track/gosling-track.ts

```diff
@@ -1377,73 +1396,10 @@ function GoslingTrack(HGC: import('@higlass/types').HGC, ...args: any[]): any {
                     return `<div>${content}</div>`;
                 }
             }
+            return '';
         }
     }
-    return new GoslingTrackClass(args);
-}
-
-const goslingIcon = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 400" width={30} height={30}>
```

Comment:
This is used anywhere in Gosling? This seems like a custom HiGlass thing, and just adds a lot of text to our library. If it's useful I can add it back, but I really don't think it's necessary.

Reply:
Removing it sounds good. As far as I know, it's useful only if we use the HiGlass modal view, but we do not use it.